apps/native-component-list/src/screens/ModulesCore/ModulesBenchmarksScreen.tsx

```diff
@@ -0,0 +1,217 @@
+import { useTheme } from 'ThemeProvider';
+import { useCallback, useState } from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+import { TurboModule, ExpoModule, BridgeModule } from '../../../../bare-expo/modules/benchmarking';
```

Comment:
maybe use [path aliases](https://docs.expo.dev/guides/typescript/#path-aliases-optional) to prevent those `../` here